index.bs

```diff
@@ -160,7 +169,15 @@ The {{PerformanceElementTiming/naturalHeight}} attribute must return the value i
 
 The {{PerformanceElementTiming/id}} attribute's getter must return <a>element</a>'s {{Element/id}} attribute value.
 
-The {{PerformanceElementTiming/element}} attribute's getter must return <a>element</a> if <a>element</a> is <a>connected</a>, and must return <code>null</code> otherwise.
+The {{PerformanceElementTiming/element}} attribute's getter must run these steps:
+<div algorithm="PerformanceElementTiming element">
+    1. Let |element| be the <a>context object</a>'s <a>element</a>.
+    1. If |element| is not <a>connected</a>, return <code>null</code>.
+    1. Let |window| be the <a>context object</a>'s <a>relevant global object</a>.
+    1. Let |document| be |window|'s <a>associated document</a>.
+    1. If |element|'s <a for="tree">root</a> is not equal to |document|, return <code>null</code>.
+    1. Return |element|.
+</div>
```

Comment:
Maybe add a note here regarding the reasoning?

Reply:
I added a note, though this just means that if the element is removed from the Document then the attribute getter no longer returns the element.

Reply:
Thanks, that makes the motivation clearer!